sdks/java/extensions/sketching/build.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: project(":").file("build_rules.gradle")
+applyJavaNature()
+
+description = "Apache Beam :: SDKs :: Java :: Extensions :: Sketching"
+
+def streamlib_version = "2.9.5"
+
+dependencies {
+  compile library.java.guava
+  shadow project(path: ":sdks:java:core", configuration: "shadow")
+  shadow "com.clearspring.analytics:stream:$streamlib_version"
+  shadow library.java.slf4j_api
+  shadow library.java.findbugs_jsr305
+  shadowTest library.java.avro
+  shadowTest library.java.commons_lang3
+  testCompile project(path: ":sdks:java:core", configuration: "shadowTest")
```

Comment:
All the `testCompile`'s should be `shadowTest`.

I wrote this block to help explain what configuration should be choosen:
https://github.com/apache/beam/blob/master/build_rules.gradle#L149
The comment also looks like it has a typo where it should say shadowTest instead of testShadow

Reply:
ok, rebasing and fixing the typo too;